packages/app/src/studio/StudioPanel.vue

```diff
@@ -10,32 +10,40 @@
   </div>
 </template>
 <script lang="ts" setup>
-import { ref, onMounted, onBeforeUnmount } from 'vue'
+import { ref, onMounted, onBeforeUnmount, watch } from 'vue'
 import { init, loadRemote } from '@module-federation/runtime'
 import type { StudioAppDefaultShape, StudioPanelShape } from './studio-app-types'
 
+const props = defineProps<{
+  canAccessStudioLLM: boolean
+}>()
+
 interface StudioApp { default: StudioAppDefaultShape }
 
 const root = ref<HTMLElement | null>(null)
 const error = ref<string | null>(null)
 const Panel = ref<StudioPanelShape | null>(null)
+const reactRoot = ref<any | null>(null)
```

Comment:
is there any way to improve the types here from `any`?

Reply:
Not easily. It would need to be a React type which doesn't seem to play nicely with the rest of the set up in the app project.